xc7/requirements.txt

```diff
@@ -1,3 +1,4 @@
 python-constraint
 git+https://github.com/symbiflow/fasm
+git+git://github.com/SymbiFlow/prjxray.git#egg=prjxray
```

Comment:
I thought we split these tools out into their own repository?

Reply:
Yes, we use xc-fasm to do `fasm -> frames -> bitstream`. The problem is that xc-fasm relies on scripts in prjxray. Unless we completely separate and re-work prjxray, we'll need to install that as a python dependency.

Reply:
You should always use `git+https://` and never `git+git`.

Reply:
> Yes, we use xc-fasm to do `fasm -> frames -> bitstream`. The problem is that xc-fasm relies on scripts in prjxray. Unless we completely separate and re-work prjxray, we'll need to install that as a python dependency.

If `xc-fasm` still requires `prjxray` why did we split it out of `prjxray`?

Reply:
> You should always use git+https:// and never git+git

I'll fix that

> If xc-fasm still requires prjxray why did we split it out of prjxray?

That was to keep things simpler. prjxray contains lots of stuff that is not related to the generation of bitstreams, whereas, having this part split makes everything better contained. prjxray is needed as a dependency as it cointains scripts to access the db, and it is used as a library.